From e92ffaf561a6370317712ece90032a662880f5ac Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 22 Mar 2017 12:44:20 +0800
Subject: [PATCH 0905/1345] Revert "arm64: clock: armada3700: add gating clock
 for eip97"

commit  b7852a8e1037a131af83c2cd596293f5aba1dbbc from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- This patch reverts commit 3160fdab8417 ("arm64: clock: armada3700:
  add gating clock for eip97").

Change-Id: I07dde4545d060644e6070dce150b3a34575c24ca
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37791
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/mvebu/armada-3700.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/clk/mvebu/armada-3700.c b/drivers/clk/mvebu/armada-3700.c
index f454138..5e9e81c 100644
--- a/drivers/clk/mvebu/armada-3700.c
+++ b/drivers/clk/mvebu/armada-3700.c
@@ -317,7 +317,6 @@ static void __init armada_3700_coreclk_init(struct device_node *np)
 	{ "spi-gate", "spi-core", 12, 0, CLK_GATE_SET_TO_DISABLE },
 	{ "twsi2", NULL, 16, 0, CLK_GATE_SET_TO_DISABLE },
 	{ "twsi1", NULL, 17, 0, CLK_GATE_SET_TO_DISABLE },
-	{ "eip97", NULL, 15, 0, CLK_GATE_SET_TO_DISABLE },
 	{ }
 };
 
-- 
1.7.9.5

